pallets/stellar-relay/src/tests.rs

```diff
@@ -448,8 +452,11 @@ fn update_tier_1_validator_set_works() {
 			public_key: Default::default(),
 			organization_id: organization.id,
 		};
-		let new_validator_set = vec![validator; 2];
-		let new_organization_set = vec![organization; 2];
+
+		let mut validator_1 = validator.clone();
+		validator_1.public_key = BoundedVec::<u8, FieldLength>::try_from(vec![1u8; 128]).unwrap();
```

Comment:
This actually boils down to `validator_1` having the same public key as previously (in line 425). So we could just clone and re-use the validator_1 we had previously I guess (?)

Reply:
Agree. Done